packages/site-components/src/SearchInput/index.tsx

```diff
@@ -22,13 +23,22 @@ export function SearchInput() {
   const handleClear = useCallback(() => {
     setSearchTerm('');
     setSearchResults([]);
+    setSearchOverflow(0);
   }, [searchTerm, searchResults]);
 
   useEffect(() => {
+    const MAX_RESULTS = 10;
```

Comment:
move this outside the component

Reply:
Done. Will be superseded by a prop soon anyway, so only a temporary const